### ISSUE TITLE
@include directive is expecting files to be relative to webpack path instead of grammar file

### ISSUE THREAD

**Issue by seqai** [kozily/nearley-loader#3]
Hello!

It is not mentioned as a feature or expected behavior in docs. Here's the case:
I have a main grammar file in e.g. `parser/main.ne`
It has a reference inside like `@include "primitives.ne"`, referencing a file `parser/primitives.ne`
Somewhere deep inside my project I have a reference to a grammar like `const  grammar = require("./../../../parsers/main.ne");`
After I run webpack I receive an error: 
```
ERROR in ./parser/main.ne
Module build failed: Error: ENOENT: no such file or directory, open 'project-path/primitives.ne'
```

If I change reference in `main.ne` to `@include "parser/primitives.ne"` then everything work nicely, and its a workaround for me, though native nearley tools are not working in this case. I think I can also change paths using webpack resolving aliases. Still it is not very convinient.

Do you think this behavior is correct or it should be changed?
Thank you!

**Comment by andres-arana** [comment]
@seqai as I said on the PR, sorry for the extreme delay on getting some feedback on this. I had some personal issues that have now been resolved, so I can get back to working on this.

I agree with your change to use relative files in include directives, mainly due to 2 reasons:

* It's the way the nearly cli compiler works, so you can use your grammars both using this loader and using the standard nearley cli tools.
* The absolute path logic breaks if you have multiple files and you want to load them both as the entrypoint to nearley in some cases and as an include dependecy in others.

I've merge your PR to branch `nearley1`, because `nearley@^2` support is on master and there were some incompatibilities there. I also applied your fix to the master branch as well.

I'll be releasing `nearley-loader@^1` and `nearley-loader@latest` to npm shortly containing this fix.

Thank you for your contribution.

**Comment by andres-arana** [comment]
Fixed in both `nearley-loader@1.0.0` and `nearley-loader@2.0.0`.
